virtru-sdk-js-drive-upload/.google/credentials.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
@trevorfoskett don't you want to add this to `.gitignore` to prevent the chance of leaking creds?